app/_src/networking/hostnamegenerator.md

```diff
@@ -2,32 +2,139 @@
 title: HostnameGenerator
 ---
 
+{% if_version gte:2.9.x %}
```

Comment:
sorry i meant the removal of the warning, the if_version should be
```suggestion
{% if_version lte:2.8.x %}
```

Reply:
duh!